src/Honeycomb.OpenTelemetry/HoneycombOptions.cs

```diff
@@ -76,6 +50,15 @@ static HoneycombOptions()
         /// </summary>
         public string ApiKey { get; set; }
 
+        /// <summary>
+        /// Returns whether <see cref="ApiKey"/> is a legacy key.
+        /// </summary>
+        internal Boolean IsLegacyKey()

```

Comment:
Your java is showing 😄 

```suggestion
        internal bool IsLegacyKey()
```